composer.json

```diff
@@ -68,13 +67,17 @@
         "symplify/vendor-patches": "^11.2",
         "tomasvotruba/class-leak": "^0.1",
         "tomasvotruba/cognitive-complexity": "^0.1",
+        "tomasvotruba/lines": "0.3.1.72",
         "tomasvotruba/type-coverage": "^0.2",
-        "tomasvotruba/unused-public": "^0.2"
+        "tomasvotruba/unused-public": "^0.2",
+        "tracy/tracy": "^2.9"
```

Comment:
all rector extensions will require-dev tracy/tracy then, or require back rector-debugging

https://github.com/rectorphp/rector-debugging

Reply:
I'm aware :+1:  Let's add tracy/tracy if needed actually :)

Reply:
I think require-dev `rector/rector-debugging` is the fastest way to avoid unnecesary add new functions.

Reply:
i want to keep dependency amount as low as possible. It's a common practice to let devs use the debugging packages/tools they need, when they need it.

Lets keep it as it is now.

Reply:
What I mean is add back `rector/rector-debugging` to require-dev into every rector-* dependencies, and add here in require-dev as well.

require-dev is not shipped in rector build, but that will speed up dev

Reply:
The issue without debug tool in rector extensions, eg: rector-phpunit is, that it will cause contributors hard to debug the node, even us, that's why if it removed, we need to add back `rector/rector-debugging` to `require-dev` in every packages:

- rector-src
- rector-phpunit
- rector-symfony
- rector-doctrine
- rector-downgrade-php

require-dev is not shipped in rector build, but that will speed up dev

Reply:
For note, https://github.com/rectorphp/rector-debugging is `archived`, if that will be used, that's need to be unarchive.

Reply:
> For note, [rectorphp/rector-debugging](https://github.com/rectorphp/rector-debugging) is archived, if that will be used, that's need to be unarchive.

That's what I want to avoid → less dependencies :) That repository was not a good idea.